migrations/2018-08-28-010336_init/up.sql

```diff
@@ -0,0 +1,70 @@
+CREATE DATABASE IF NOT EXISTS `syncstorage` /*!40100 DEFAULT CHARACTER SET latin1 */;
```

Comment:
The BSO payload can in theory be arbitrary unicode, since it's JSON.  Should we set an explicit encoding like `utf8mb4` to guard against any weirdness in unicode handling?